storage/innobase/xtrabackup/test/t/innodb_keyring_file_component.sh

```diff
@@ -0,0 +1,61 @@
+#
+# Component keyring file specific variables
+#
+require_server_version_higher_than 8.0.23
+
+KEYRING_TYPE="component"
+. inc/keyring_common.sh
+. inc/keyring_file.sh
+configure_server_with_component
+
+run_cmd $MYSQL $MYSQL_ARGS test <<EOF
+CREATE TABLE t1 (c1 VARCHAR(100)) ENCRYPTION='y';
+INSERT INTO t1 (c1) VALUES ('ONE'), ('TWO'), ('THREE');
+INSERT INTO t1 (c1) VALUES ('10'), ('20'), ('30');
+INSERT INTO t1 SELECT * FROM t1;
+INSERT INTO t1 SELECT * FROM t1;
+INSERT INTO t1 SELECT * FROM t1;
+INSERT INTO t1 SELECT * FROM t1;
+ALTER INSTANCE ROTATE INNODB MASTER KEY;
+CREATE TABLE t2 (c1 VARCHAR(100)) ENCRYPTION='y';
+INSERT INTO t2 SELECT * FROM t1;
+EOF
+
+xtrabackup --backup --target-dir=$topdir/backup1
```

Comment:
Please add a test scenarios with --component-ekyring-file-config passed to backup
1. passing a valid file
2. passing empty without value
3. passing rubbish file

Reply:
@satya-bodapati for backup this is not an option. Backup uses the online function and whatever is loaded into the server is the source of truth.

Reply:
Right, u could say negative testing :wink:

Reply:
@satya-bodapati 

```
marcelo@marce-bld:/work/pxb/ins/8.0/xtrabackup-test/test$ ./run.sh -t t/innodb_keyring_file_component.sh
Using /work/pxb/ins/8.0/xtrabackup-test/test/var as test root
Detecting server version...
Running against MySQL 8.0.25 (InnoDB 8.0.25)
Using 'xtrabackup' as xtrabackup binary
Autodetected number of cores: 32
Limiting to 16 to avoid excessive resource consumption
Using 16 parallel workers

==============================================================================
TEST                                   WORKER    RESULT     TIME(s) or COMMENT
------------------------------------------------------------------------------
innodb_keyring_file_component            w1	[passed]    18
==============================================================================
Spent 18 of 25 seconds executing testcases

SUMMARY: 1 run, 1 successful, 0 skipped, 0 failed

See results/ for detailed output
marcelo@marce-bld:/work/pxb/ins/8.0/xtrabackup-test/test$ grep "\-\-component-keyring-file-config will be ignored for \-\-backup operation" results/innodb_keyring_file_component
xtrabackup: Warning: --component-keyring-file-config will be ignored for --backup operation
marcelo@marce-bld:/work/pxb/ins/8.0/xtrabackup-test/test$
```

Reply:
Thanks, I will check the updated test